pyproject.toml

```diff
@@ -23,6 +23,9 @@ max_line_length = 99
 addopts = "-rsx --tb=short"
 testpaths = ["tests"]
 filterwarnings = "always"
+markers = [
+    "network",
```

Comment:
@agronholm The marker should be declared here.

Reply:
Right; I'm not sure why I got this warning the other time. Not seeing it now. Thanks.